### PR TITLE
add deletePlayerScore for LL&LLSE

### PR DESCRIPTION
**Pull request by kkj9333**
## Description
The format change is due to the clang's ctrl+K and ctrl+D

PR DESCRIPTION
add deletePlayerScore for LL&LLSE
adjust offlineScoreAPI
## Issues fixed
Wait for a lucky man to review...

_None_

### CODE REVIEW

**Review by Redbeanw44602 — CHANGES_REQUESTED** [review]
Please avoid modifying the content of files that are not related to the PR content as much as possible, please roll back the format.

**Comment by kkj9333** [comment]
> Please avoid modifying the content of files that are not related to the PR content as much as possible, please roll back the format.

However, the format change is caused by the clang-format provided by ll. If the format change is caused, it can only indicate that there are some format problems in the original code. So should I ignore these and roll back?

File: LiteLoader/include/llapi/mc/Scoreboard.hpp [hunk 1]
```diff
@@ -218,16 +247,19 @@ class Scoreboard {
     virtual ~Scoreboard();
     /**
      * @vftbl  1
-     * @symbol ?setDisplayObjective@Scoreboard@@UEAAPEBVDisplayObjective@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBVObjective@@W4ObjectiveSortOrder@@@Z
+     * @symbol
```
Comment (ShrBox) [inline LiteLoader/include/llapi/mc/Scoreboard.hpp:250]: 不要将由HeaderGenerator生成的头文件的注释进行格式化，否则会造成CPP文档无法正常生成

Comment (kkj9333) [reply LiteLoader/include/llapi/mc/Scoreboard.hpp:250]: ok

**Pull request closed without merging**